diff --git a/dist/CommitParser.js b/dist/CommitParser.js
index b63ffb1e7259e4c0194ceb2933f536f24ed9597e..3b75f888ff5f202041183b694073b37b7e61b26c 100644
--- a/dist/CommitParser.js
+++ b/dist/CommitParser.js
@@ -74,6 +74,9 @@ export class CommitParser {
     }
     parseReferences(input) {
         const { regexes } = this;
+        if (input == null) {
+            return [];
+        }
         const regex = input.match(regexes.references)
             ? regexes.references
             : /()(.+)/gi;
